package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "@4c/layout",
-  "version": "1.0.0",
-  "main": "lib/index.js",
-  "modules": "es/index.js",
-  "jsnext:main": "es/index.js",
+  "version": "0.0.0-development",
+  "main": "lib/css-modules/index.js",
```

Comment:
why the namespacing?

Reply:
in my head there may be multiple versions per styling target here, like a Sass only implementation

Reply:
or styled component idk

Reply:
fair enough